src/backend/distributed/executor/local_executor.c

```diff
@@ -909,6 +909,33 @@ TaskAccessesLocalNode(Task *task)
 }
 
 
+#if DELETE
+
+/*
+ * TaskAccessesRemoteNode returns true if any placements of the task reside on
+ * a remote node other than that we're executing the query.
+ */
+bool
+TaskAccessesRemoteNode(Task *task)
```

Comment:
this function seems unused

Reply:
Yeah, we don't need it anymore, removed.